frontend/src/Components/NAVBAR.js

```diff
@@ -0,0 +1,101 @@
+import React from "react";
+import "../navbar.css";
```

Comment:
component specific styles should be in same dir as component, try storing navbar in `components/navbar/` for sanity. PRØTIP: name navbar.js file `index.js` and import it like `import Navbar from 'components/navbar'`